main/zoe/api/zoe-helpers.md

```diff
@@ -3,324 +3,241 @@
 ZoeHelpers are functions that extract common contract code and
 patterns into reusable helpers.
 
-These helper functions can be imported from @agoric/zoe/src/contractSupport. We
-expect to move them to a separate package shortly, so it would become
-'@agoric/zoe-contract-support'. The import provides a function `makeZoeHelpers()`,
-which produces versions of the function that are bound to the current zoe instance.
-
+All of the ZoeHelper methods are described below. To use any of them, import
+it directly from `@agoric/zoe/src/contractSupport/`. For example, the following 
+imports the two ZoeHelpers `assertIssuerKeywords()` and `assertProposalShape()`:
 ```js
-import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
-
-const {
-  assertKeywords,
-  assertNatMathHelpers,
-  checkIfProposal,
-  rejectOffer,
-  swap,
-  rejectIfNotProposal,
-  getActiveOffers,
-  makeEmptyOffer,
-  escrowAndAllocateTo,
-  isOfferSafe,
-  satisfies,
-  trade,
-} = makeZoeHelpers(zoe);
+import {
+  assertIssuerKeywords,
+  assertProposalShape,
+} from '@agoric/zoe/src/contractSupport';
 ```
 
-## zoeHelpers.assertKeywords(keywords)
-- `keywords` `{Array <String>}`
+Note that almost all ZoeHelpers require `zcf`, a `ContractFacet` as a first argument.
+Since they are contract helpers, they have access to the contract source code, which
+passes in `zcf`, the contract facet object, when it runs `start()`. 
 
-Checks that the keywords submitted by the creator of the contract
-instance match what the contract expects. Throws if incorrect or if there is
-missing or extra keywords. Order of keywords is irrelevant.
+## assertIssuerKeywords(zcf, keywords)
+- `zcf` `{ContractFacet}` 
+- `keywords` `{Array <String>}`
 
-```js
-import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
+This method always takes `zcf` as its first argument.
 
-const { assertKeywords } = makeZoeHelpers(zoe);
+Checks that keywords submitted by the contract instance creator
+match what the contract expects. Throws if incorrect or if there are
+missing or extra keywords. The keywords order is irrelevant.
 
-// proposal for offerHandle
-const proposal = {
-  want: { Asset: moola(4) },
-  give: { Price: simoleans(16) },
-  exit: { onDemand: null },
-}
+```js
+import {
+  assertIssuerKeywords,
+} from '@agoric/zoe/src/contractSupport';
 
-assertKeywords(['Asset', 'Price']);
+// proposals for this contract instance use keywords 'Asset' and 'Price'
+assertIssuerKeywords(zcf, harden(['Asset', 'Price']));
 ```
 
-## zoeHelpers.assertNatMathHelpers(brand)
-- `brand` `{String}`
-
-Given a brand, assert that the mathHelpers for that issuer
-are the 'nat' type (i.e. that the issuer is associated with
-fungible assets). Returns `true` if they are `nat`, `false` if
-they are not.
+## assertUsesNatMath
+- `zcf`- `{ContractFacet}`
+- `brand` - `{Brand}`
 
+This method always takes `zcf` as its first argument. 
```

Comment:
It's a combination of needing to say something about it and having the same information repeated in each applicable method description since a lot of the usage of this doc will be going to one specific method, rather than reading in sequence.